app/sidecar/app.toml

```diff
@@ -214,10 +214,14 @@ task-slots = ["sys", "i2c_driver", {spi_driver = "spi1_driver"}]
 name = "drv-sidecar-seq-server"
 features = []
 priority = 3
-max-sizes = {flash = 131072, ram = 2048}
+max-sizes = {flash = 262144, ram = 2048}
```

Comment:
No need to change anything for this PR, but I wonder if this is now big enough to make `miniz` worthwhile 👀...

(It wasn't worth it for Gimlet, because it saved 10K of FPGA image size and added 11K of decompressor code)

Reply:
Maybe? I think we soon need to do the work to enable the additional flash we have on this board to store these images. That'll free up this as well as other init blobs.